css-grid-classes/style.css

```diff
@@ -0,0 +1,45 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+h2 {
+  height: 15%;
+}
+
+.column-link-yoshi {
+  font-size: 10px;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.text {
+  display: inline-block;
+  font-size: 15px;
+  font-weight: bold;
+}
+
+.column-mario-dk {
+  width: 800px;
+  display: flex;
+  justify-content: space-around;
+}
+
+.pics > img {
+  margin-left: auto;
+  width: 150px;
+  display: flex;
+  justify-content: left;
+}
+
+.column-four-items {
+  width: 800px;
+  display: flex;
```

Comment:
The flexbox properties should be unnecessary here. Also, same not as above, use a responsive unit rather than pixels 👍